webui/common/package.json

```diff
@@ -27,17 +27,13 @@
     "start-js": "BROWSER=none react-scripts start --watch --recursive",
     "build-js": "react-scripts build --production",
     "start": "npm-run-all --parallel watch-css start-js",
-    "build": "NODE_ENV=production && npm-run-all --parallel build-css build-js",
+    "build": "eslint 'src/**/*.tsx' && NODE_ENV=production && npm-run-all --parallel build-css build-js",
```

Comment:
Is there a reason to set 
```
NODE_ENV=production && npm-run-all ...`
```
vs 
```
NODE_ENV=production npm-run-all ...
```

I think the ampersand is unnecessary?

Reply:
updated